src/awkward/_v2/_connect/numpy.py

```diff
@@ -34,8 +34,6 @@ def convert_to_array(layout, args, kwargs):
 #             ak._v2.contents.Content,
 #             ak._v2.record.Record,
 #             ak.layout.ArrayBuilder,
-#             ak.layout.LayoutBuilder32,
-#             ak.layout.LayoutBuilder64,
```

Comment:
Perhaps, it's worth deleting the whole commented out code block?

Reply:
That's what I would usually want to do, but since this is a legacy version, I just don't want to change much.